### DISCUSSION TITLE
App Questions

### DISCUSSION THREAD

**Discussion by DJShocker** [OwlGamingCommunity/MTA#19] [Support]
Upon registration, this happens:
[21-12-25 00:53:42] ERROR: Infinite/too long execution (apps)
[21-12-25 00:53:42] ERROR: Aborting; infinite running script in apps

No questions pop up, i do have some test questions in. The console says it occurs around line 41 of  app_step1_s.lua

I have 4 questions in, just generic dummy questions. am I missing something?

*EDIT* Disregard, I misread it and each step requires 6 questions. Oops.

**Comment by Pipo1978** [comment]
> Upon registration, this happens: [21-12-25 00:53:42] ERROR: Infinite/too long execution (apps) [21-12-25 00:53:42] ERROR: Aborting; infinite running script in apps
> 
> No questions pop up, i do have some test questions in. The console says it occurs around line 41 of app_step1_s.lua
> 
> I have 4 questions in, just generic dummy questions. am I missing something?
> 
> _EDIT_ Disregard, I misread it and each step requires 6 questions. Oops.

Hey mate, how many questions you wrote for each step? I'm stuck on "step2_s"

_[00:58:53] ERROR: apps/app_step2_s.lua:23: Aborting; infinite running script in apps_

I've create more 6 dummy questions and set "part" table on 2, but still face this issue.

Thanks ;)

**Comment by mohamedbenhandil** [comment]
Hey did you found the solution i'm stuck with the same think, no table

**Comment by ssoheilnz** [comment]
جونم دادا مشکلت چیه
